.github/workflows/build.yml

```diff
@@ -124,30 +124,3 @@ jobs:
         with:
           name: ${{ github.sha }}
           path: target/release/*-Windows-x86_64.exe
-
```

Comment:
We only want to run "test docs build" during `CI`, not `build`, as we dont care about uploading those artifacts anywhere anymore.